.github/workflows/ubuntu-arm.yml

```diff
@@ -0,0 +1,84 @@
+name: Build-Vulkan-SDK-ARM
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Vulkan SDK version (e.g., 1.4.314.0)'
+        required: false
+        default: 'latest'
+
+jobs:
+  build-linux-arm:
+    name: Build Vulkan-SDK-ARM for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-24.04-arm, ubuntu-22.04-arm ]
```

Comment:
Do we need to run on 2 ubuntu versions? For the purposes of building, I guess 1 is enough? Or ... what advantage are you seeing in running the build on 2 different ubuntu versions?

Reply:
> Do we need to run on 2 ubuntu versions? For the purposes of building, I guess 1 is enough? Or ... what advantage are you seeing in running the build on 2 different ubuntu versions?

you can choice under dispatch, but it is prefer with 22.04